doc/nrf/releases/release-notes-1.7.0.rst

```diff
@@ -0,0 +1,601 @@
+
+Other samples
+-------------
+
+* :ref:`radio_test` - Added an automatic build of the :ref:`nrf5340_empty_app_core` sample, when building for ``nrf5340dk_nrf5340_cpunet``.
+
+Common
+======
+
+The following changes are relevant for all device families.
+
+sdk-nrfxlib
+-----------
+
+* Updated the default :ref:`nrf_rpc` transport backend to use the RPMsg Service library.
+
+See the changelog for each library in the :doc:`nrfxlib documentation <nrfxlib:README>` for additional information.
+
+Modem library
++++++++++++++
+
+* Added:
+
+  * Added a new API for AT commands.
+    See :ref:`nrfxlib:nrf_modem_at` for more information.
+  * Added a new API for modem delta firmware updates.
+    See :ref:`nrfxlib:nrf_modem_delta_dfu` for more information.
+
+* Updated:
+
+  * Updated :ref:`nrf_modem` to version 1.3.0.
+    See the :ref:`nrfxlib:nrf_modem_changelog` for detailed information.
+
+* Deprecated:
+
+  * The AT socket API is now deprecated.
+  * The DFU socket API is now deprecated.
+
+NFC
+---
+
+* Updated the NFCT interrupt configuration to reduce the ISR latency and enable scheduling decision in the interrupt context.
+* Updated the :ref:`nfc_uri` library to allow encoding of URI strings longer than 255 characters.
+
+Trusted Firmware-M
+------------------
+
+* Added a test case for the secure read service that verifies that only addresses within the accepted range can be read.
+* Updated :file:`tfm_platform_system.c` to fix a bug that returned ``TFM_PLATFORM_ERR_SUCCESS`` instead of ``TFM_PLATFORM_ERR_INVALID_PARAM`` when the address passed is outside of the accepted read range.
+
+Common Application Framework (CAF)
+----------------------------------
+
+* Added :ref:`caf_net_state`.
+* Added :ref:`caf_power_manager`.
+* Updated :ref:`caf_sensor_sampler` with a limit to the number of ``sensor_event`` events that it submits.
+
+Profiler
+--------
+
+* Added profiling string data.
+* Optimized numeric data encoding.
+
+Edge Impulse
+------------
+
+* Added support for Thingy:53 to :ref:`nrf_machine_learning_app`.
+* Added configuration for nRF52840 DK that supports data forwarder over NUS to :ref:`nrf_machine_learning_app`.
+
+Pelion
+------
+
+* Updated Pelion Device Management Client library version to 4.10.0.
+* Switched to using :ref:`caf_power_manager` and :ref:`caf_net_state` in :ref:`pelion_client`.
+* Updated the :ref:`application documentation <pelion_client>` with a step that requires downloading Pelion development tools.
+
+Build system
+------------
+
+* Bugfixes:
+
+  * Fixed a bug where :file:`dfu_application.zip` would not be updated after rebuilding the code with changes.
+
+MCUboot
+=======
+
+The MCUboot fork in |NCS| (``sdk-mcuboot``) contains all commits from the upstream MCUboot repository up to and including ``7a51968``, plus some |NCS| specific additions.
```

Comment:
@sigvartmh , @hakonfam , can you confirm this is the correct commit ID?

Reply:
@SebastianBoe to confirm

Reply:
Was the wrong commit. Fixed now.